content/agent/kubernetes/event_collection.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Kubernetes Event Collection
+kind: documentation
+---
+
+Similarly to Agent 5, Agent 6 can collect events from the Kubernetes API server.
+Set the `collect_kubernetes_events` variable to `true` in the datadog.yaml. This can be achieved via the environment variable `DD_COLLECT_KUBERNETES_EVENTS` that is resolved at start time.
+You will need to give the Agent some rights to activate this feature: see the [RBAC][1]section.
+
+A [ConfigMap][3] can be used to store the `event.tokenKey` and the `event.tokenTimestamp`. It has to be deployed in the `default` namespace and be named `datadogtoken`.
```

Comment:
Sorry for the delay - Indeed, this is not true anymore. The resources have to be deployed in the same namespace however. It can be configured with `DD_KUBE_RESOURCES_NAMESPACE`.
https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L218